### DISCUSSION TITLE
Welcome to comitup Discussions!

### DISCUSSION THREAD

**Discussion by davesteele** [davesteele/comitup#131] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  We're going to try this out as a possibly useful feature i.e. - it may go away.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GerritRie** [comment]
Hey Dave, I have a huge Problem. Can u Maybe give me hand ?
best Regards
